Python/specialize.c

```diff
@@ -355,6 +355,28 @@ static int function_kind(PyCodeObject *code);
 static bool function_check_args(PyObject *o, int expected_argcount, int opcode);
 static uint32_t function_get_version(PyObject *o, int opcode);
 
+#ifdef Py_GIL_DISABLED
+static void
+maybe_enable_deferred_ref_count(PyObject *dict, PyObject *name)
+{
+    PyObject *op;
+    if (PyDict_GetItemRef(dict, name, &op) != 1) {
+        return;
+    }
```

Comment:
Let's combine this lookup with the earlier `_PyDict_LookupIndex`. `_PyDict_LookupIndex` already gets the value internally and throws it away.